internal/pkg/cli/app_show.go

```diff
@@ -257,7 +257,7 @@ func BuildAppShowCmd() *cobra.Command {
 	}
 	cmd := &cobra.Command{
 		Use:   "show",
-		Short: "Displays information about an application per environment.",
+		Short: "Show details of an application per environment.",
```

Comment:
Yeah I agree that "details" might not be good enough since we don't show any live status info. How about "settings" (since we only show static information before deploying)?